### ISSUE TITLE
Deal with upstream docker-boshrelease changes

### ISSUE THREAD

**Issue by mboldt** [cf-platform-eng/tile-generator#181]
It looks like the new [bosh2-ified docker-boshrelease](https://github.com/cloudfoundry-community/docker-boshrelease/releases/tag/v30.0.0) broke stuff. Here is the beginning of the string of errors from CI:

```
  Error 100: Unable to render instance groups for deployment. Errors are:
     - Unable to render jobs for instance group 'docker-bosh-tg_test_app4'. Errors are:
       - Error filling in template '(unknown)' (line (unknown): containers/monit:1: syntax error, unexpected ';'
      ...rs', []).each do |container| -; _erbout.concat "\ncheck proc...
      ...                               ^
      containers/monit:7: syntax error, unexpected ';'
      ;  end -; _erbout.concat "\n"
               ^)
```

I plan to pin to docker-boshrelease 29.0.0 in the meantime.

**Comment by drnic** [comment]
Sorry about that. But your resulting tile metadata files will be more lovely once you move to it!

A snippet from Dingo Redis metadata is:

```yaml
job_types:
- name: docker
  resource_label: Docker Backend
  templates:
  - name: docker
    release: docker
    manifest: |
      tcp_address: 0.0.0.0
      tls_cacert: (( $ops_manager.ca_certificate ))
      tls_cert: ((.properties.docker_tls.cert_pem))
      tls_key: ((.properties.docker_tls.private_key_pem))

property_blueprints:
- name: docker_tls
  type: rsa_cert_credentials
```

(Note: `tcp_address: 0.0.0.0` is probably optional; it allows remote docker clients to connect to the docker daemon which might be helpful for docker-orientation tooling/discovery)

**Comment by drnic** [comment]
For running containers, here is a snippet for the `containers` job: https://github.com/cloudfoundry-community/docker-boshrelease/blob/master/manifests/containers/example.yml#L20-L41

**Comment by mboldt** [comment]
Thanks, @drnic. Please, no need to apologize! This is a change for the better.

**Comment by mboldt** [comment]
This was resolved in 8868772aa5620dc203a37d2e49b1101cf5229b08. Yay!
